Prefabs/Locomotion/BodyRepresentation/SharedResources/Scripts/BodyRepresentationInternalSetup.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace VRTK.Core.Prefabs.Locomotion.BodyRepresentation
+{
+    using UnityEngine;
+    using VRTK.Core.Data.Type;
+    using VRTK.Core.Prefabs.Locomotion.Teleporters;
+    using VRTK.Core.Tracking;
+    using VRTK.Core.Tracking.CharacterController;
+    using VRTK.Core.Tracking.Modification;
+
+    /// <summary>
+    /// Sets up the BodyRepresentation Prefab based on the provided user settings.
+    /// </summary>
+    public class BodyRepresentationInternalSetup : MonoBehaviour
+    {
+        [Header("Facade Settings")]
```

Comment:
Inline these as per contributing guidelines, plus wrap section in `#region <Header Name>` tag